src/ripplegw/interfaces.py

```
@@ -612,6 +613,61 @@ def __repr__(self):
+        """
+        Args:
+            f_ref (float): Reference frequency in Hz. Defaults to 20.0.
+        """
+        self.f_ref = f_ref
+
+    @property
+    def parameter_names(self) -> tuple[str, ...]:
+        return ("M_c", "eta", "s1_z", "s2_z", "d_L", "phase_c", "iota")
+
+    def __call__(
+        self, frequency: Float[Array, " n_freq"], params: dict[str, Float]
```

Ruff (0.15.12)

[error] Syntax error in forward annotation: Expected an expression

(F722)